src/main/java/com/trailfinder/dao/EventDAOStub.java

```diff
@@ -3,7 +3,7 @@
  */
 package com.trailfinder.dao;
 
-import java.util.List;
+//import java.util.List;
```

Comment:
Don't commit commented out code.  This is a no-no in version control, because you can always go back in history to see what used to be there.